internal/handlers/handlers_suite_test.go

```diff
@@ -20,6 +20,8 @@ func TestHandlers(t *testing.T) {
 
 func createTestISO() string {
 	filesDir, err := os.MkdirTemp("", "isotest")
+	initrdAddrsize := []byte{
```

Comment:
what do these bytes mean exactly?

Reply:
I choose the values for testing purpose to make it easier to debug and verify. If you checking the ISO having the initrd.addrsize it contains the values based on the kernel.img and initrd.img (the loading address of the initrd.img depends on the size of the kernel alligned to MiB boundary to get the PSW for the initrd).

Reply:
Are they arbitrary though? if they have some sort of meaning/format, please use something like e.g.:

```go
bs := make([]byte, 8)
binary.LittleEndian.PutUint64(bs, uint64(64))
```

 To generate them